packages/auth/src/accessToken/service.test.ts

```diff
@@ -1,7 +1,7 @@
 import { faker } from '@faker-js/faker'
 import nock from 'nock'
 import { Knex } from 'knex'
-import { v4 } from 'uuid'
+import { v4 as uuid } from 'uuid'
```

Comment:
Consistency matters! 😂

Reply:
I regretted this change as soon as I saw it in the diff 😆